docs/content/main/magic-vars.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Magic Variables"
+slug = "magic variables documenation"
+weight = 3
++++
+
+# Magic Variables
+These are variables that are always there, without you having to import them.
+
+The delay between the updating variables is always 2s.
+
+
+## Updating magic variables
+
```

Comment:
This file is,... missing stuff 👀 specifically, missing the actual variables - and idk what `updating magic variables` was even supposed to be

Reply:
Ahhhhhh that's getting autogenerated, nevermind 👀